### DISCUSSION TITLE
Discord wallbash

### DISCUSSION THREAD

**Discussion by RookiexCookie** [prasanthrangan/hyprdots#931] [Ideas]
hey @kRHYME7 i know you probably busy with the kitty wallbash thingy but hear me out
So i was wondering how to get the discord be the same theme as the rest of theme, so i started researching and found this thing https://github.com/danihek/Themecord
its a css file which changes the theme of the vencord installed discord using pywall but thats not the point, the css file got some easy defined color variables  like this 
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/cefe4734-8cb8-44df-bac0-2ab7bbda2c54)
so i copied the css file to the theme folder and i manually changed the color of the stuff using Wall.dcol of kitty as a reference as i dont know how to use yk wallbash with scripts and all that anyway so this is the result that i got
![240221_19h00m23s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/136296017/500f0914-f6ce-4ea8-8b5d-30733c9b0958)

**Comment by kRHYME7** [comment]
> hey @kRHYME7 i know you probably busy with the kitty wallbash thingy but hear me out 

Yeah, almost done, ended up overriding different terminals, e.g alacritty,konsole,etc which sounds like good news. 

>  wallbash with scripts and all that anyway so this is the result that i got

WOW !  I'll try to incorporate this on my free time, it should be easy I think. 

Can you share the .dcol file or the CSS file, whichever you have.  Thanks!

**Comment by RookiexCookie** [comment]
oh great work , getting them all work together
and uh 
i just download the css file from that github page that i linked in the above message as the github is saying they dont support css file so i cant send you directly here.[the name of the file is Themecord.css]
But all i did was copy pasted the colors from other dcol file , anyways
for discord to have this theme they have to install the vencord mod and also the location for themes is `~/.config/Vencord/themes/` there the css files should be at so vencord can see it.

**Reply by kRHYME7** [reply]
So we need vencord to pass those colors to _discord?_ or Discord will read inside this files. Did not read their docu yet, but should all of this CLI base? no gui interaction involve right?

**Comment by RookiexCookie** [comment]
Yup we need the vencord as official discord client doesnt support custom css (like for spotify there is spicetify ) anyways yes when i  changed the colors values  in vscode/terminal after saving the changes and  switched to the discord workspace , it automatically reloaded the new theme and all done with codes no gui was needed

**Reply by akashgagda** [reply]
Hey bro did you try armcord

**Reply by RookiexCookie** [reply]
mhm yes ive tried , whyd you ask ?

**Reply by akashgagda** [reply]
It's have inbuilt vencord or css theming or client is open source

**Reply by RookiexCookie** [reply]
yes but not everyone using that , some maybe using vesktop like me which was made by the vencord dev 
anywho we will just need to check the location of theme folder respective of their discord client
to get it working , the main thing is getting the wallbash color and adding in the css file

**Reply by kRHYME7** [reply]
Can I ask what is the consensus of both armcord and vencord? Or are there any client aside from these both.  I'll try to see what we can do to vencord BTW.

**Reply by prasanthrangan** [reply]
this looks cool, i only use discord in browser... so do let me know which client to use and i'll integrate it!

**Reply by kRHYME7** [reply]
Vencord

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/f1786a59-54a7-413c-a1d9-339c6eff2108)

**Reply by akashgagda** [reply]
Vencord is good

**Reply by Enigma1309** [reply]
I really like vencord, but i'm a webcord user instead, i think the script can be extended to be used with webcord too
Webcord supports the theme.css by  default in ~/.config/WebCord/theme.css (name can be anything as soon as it ends with .theme.css)
More about webcord theming [here](https://github.com/SpacingBat3/WebCord/blob/master/docs/Features.md)

**Reply by Enigma1309** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/94917588/a8a5a1d2-31a1-4d49-8ea2-a80427888d94)
something like this can be achived , and the webcord updates automatically as soon as it detects the change of the css file

**Reply by RookiexCookie** [reply]
Alrighty , I guess they all work in same manner but only concern would be the file location, they seem to differ based on the apps
well thats doable

**Reply by kRHYME7** [reply]
Let's just hope that they use the same variables.

**Reply by Enigma1309** [reply]
yep i think they do, that's why you're current discord.dcol is working fine
